tree/30_generic_methods/file_from_shared_folder.cf

```diff
@@ -51,7 +51,7 @@ bundle agent file_from_shared_folder(source, destination, hash_type)
   methods:
       "disable_reporting_${class_prefix}"
                                usebundle => disable_reporting;
-      "copy without recursion" usebundle => file_copy_from_remote_source_recursion("/var/rudder/configuration-repository/shared-files/${source}", "${destination}", "0");
+      "copy without recursion" usebundle => file_copy_from_remote_source_recursion("/var/rudder/configuration-repository/shared-files/${source}", "${destination}", "inf");
```

Comment:
you hsould change the full inner class prefix to put 
 "full_inner_class_prefix" string => canonify("file_from_remote_source_${report_param}_*inf*");